cd-cli/cnf-templates/complete-pipeline.yaml

```diff
@@ -368,37 +368,22 @@ Resources:
                   if ( [ "$PIPELINE_NAME" = "pn-frontend-update-pipeline" ] ) then
                     echo "Check pn-frontend build end"
                     
-                    pfSvil=$( ( aws s3api head-object --bucket ${CiArtifactBucketName} --key "pn-frontend/commits/${COMMIT_ID}/pn-personafisica-webapp_svil.tar.gz" 2> /dev/null > /dev/null ) && echo "OK"  || echo "KO" )
-                    echo "pfSvil is build ${pfSvil}"
+                    pf=$( ( aws s3api head-object --bucket ${CiArtifactBucketName} --key "pn-frontend/commits/${COMMIT_ID}/pn-personafisica-webapp.tar.gz" 2> /dev/null > /dev/null ) && echo "OK"  || echo "KO" )
```

Comment:
Ho aggiunto questo file perché va modificato anche questo.
Le modifiche che ho fatto vanno bene in caso la build sia unifica per tutti gli ambienti ma ancora paralelizzata per ogni "applicazione".

@ZorbaDimatteo se mi confermi che è stata completamente eliminata la parallelizzazione delle build per il front-end allora devo rivedere questo file (semplicemente rimuovere un workaround che era stato introdotto per la gestione delle build parallele)

Reply:
@marco-vit-pagopa l'obiettivo è eliminare completamente la parallelizzazione delle build, quindi sì. c'è da attendere anche la chiusura di alcune attività su pn-frontend